### ISSUE TITLE
Strapi application breaks after installing graphql

### ISSUE THREAD

**Issue by kasonde** [strapi/strapi#11977]
### Describe the bug

My Strapi application breaks each time I install the GraphQL plugin. I know this may not be the case for everyone, but just in case it is, I'm putting the information here.

### Steps to reproduce the behavior

1. Create a blank application
2. Install the plugin using `yarn strapi install graphql`
3. Start the development server

### Code snippets (Stack trace)

```
yarn run v1.22.10
$ strapi develop

<--- Last few GCs --->

[26733:0x150008000]     1043 ms: Scavenge 62.5 (83.6) -> 52.3 (84.3) MB, 2.4 / 0.0 ms  (average mu = 0.994, current mu = 0.994) allocation failure 
[26733:0x150008000]     1073 ms: Scavenge 63.6 (84.8) -> 51.7 (85.3) MB, 0.8 / 0.0 ms  (average mu = 0.994, current mu = 0.994) task 
[26733:0x150008000]     1143 ms: Scavenge 68.1 (86.6) -> 54.4 (87.2) MB, 1.6 / 0.0 ms  (average mu = 0.994, current mu = 0.994) allocation failure 


<--- JS stacktrace --->

FATAL ERROR: wasm code commit Allocation failed - process out of memory
 1: 0x1049899d8 node::Abort() [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 2: 0x104989b58 node::errors::TryCatchScope::~TryCatchScope() [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 3: 0x104aa3330 v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 4: 0x104aa32c4 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 5: 0x105038cbc v8::internal::wasm::WasmCodeAllocator::AllocateForCodeInRegion(v8::internal::wasm::NativeModule*, unsigned long, v8::base::AddressRegion, v8::internal::wasm::WasmCodeAllocator::OptionalLock const&) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 6: 0x105039950 v8::internal::wasm::NativeModule::CreateEmptyJumpTableInRegion(int, v8::base::AddressRegion, v8::internal::wasm::WasmCodeAllocator::OptionalLock const&) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 7: 0x105038618 v8::internal::wasm::NativeModule::AddCodeSpace(v8::base::AddressRegion, v8::internal::wasm::WasmCodeAllocator::OptionalLock const&) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 8: 0x105039760 v8::internal::wasm::NativeModule::NativeModule(v8::internal::wasm::WasmEngine*, v8::internal::wasm::WasmFeatures const&, v8::internal::VirtualMemory, std::__1::shared_ptr<v8::internal::wasm::WasmModule const>, std::__1::shared_ptr<v8::internal::Counters>, std::__1::shared_ptr<v8::internal::wasm::NativeModule>*) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
 9: 0x10503bcd4 v8::internal::wasm::WasmCodeManager::NewNativeModule(v8::internal::wasm::WasmEngine*, v8::internal::Isolate*, v8::internal::wasm::WasmFeatures const&, unsigned long, std::__1::shared_ptr<v8::internal::wasm::WasmModule const>) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
10: 0x1050477e8 v8::internal::wasm::WasmEngine::NewNativeModule(v8::internal::Isolate*, v8::internal::wasm::WasmFeatures const&, std::__1::shared_ptr<v8::internal::wasm::WasmModule const>, unsigned long) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
11: 0x105014110 v8::internal::wasm::CompileToNativeModule(v8::internal::Isolate*, v8::internal::wasm::WasmFeatures const&, v8::internal::wasm::ErrorThrower*, std::__1::shared_ptr<v8::internal::wasm::WasmModule const>, v8::internal::wasm::ModuleWireBytes const&, v8::internal::Handle<v8::internal::FixedArray>*) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
12: 0x10504463c v8::internal::wasm::WasmEngine::SyncCompile(v8::internal::Isolate*, v8::internal::wasm::WasmFeatures const&, v8::internal::wasm::ErrorThrower*, v8::internal::wasm::ModuleWireBytes const&) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
13: 0x10506b344 v8::(anonymous namespace)::WebAssemblyModule(v8::FunctionCallbackInfo<v8::Value> const&) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
14: 0x104b0c040 v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
15: 0x104b0b3a4 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<true>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
16: 0x104b0aea0 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
17: 0x105220fcc Builtins_CEntry_Return1_DontSaveFPRegs_ArgvOnStack_BuiltinExit [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
18: 0x1051b577c Builtins_JSBuiltinsConstructStub [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
19: 0x10529df0c Builtins_ConstructHandler [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
20: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
21: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
22: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
23: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
24: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
25: 0x10904e10c 
26: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
27: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
28: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
29: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
30: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
31: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
32: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
33: 0x10904e10c 
34: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
35: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
36: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
37: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
38: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
39: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
40: 0x10904e10c 
41: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
42: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
43: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
44: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
45: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
46: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
47: 0x10904e10c 
48: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
49: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
50: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
51: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
52: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
53: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
54: 0x10904e10c 
55: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
56: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
57: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
58: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
59: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
60: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
61: 0x10904e10c 
62: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
63: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
64: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
65: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
66: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
67: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
68: 0x10904e10c 
69: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
70: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
71: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
72: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
73: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
74: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
75: 0x10904e10c 
76: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
77: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
78: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
79: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
80: 0x1051b562c Builtins_JSConstructStubGeneric [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
81: 0x10529e15c Builtins_ConstructWithSpreadHandler [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
82: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
83: 0x1051b2a04 Builtins_ArgumentsAdaptorTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
84: 0x1051b55fc Builtins_JSConstructStubGeneric [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
85: 0x10529df0c Builtins_ConstructHandler [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
86: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
87: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
88: 0x1051b9c34 Builtins_InterpreterEntryTrampoline [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
89: 0x1051e6b20 Builtins_AsyncFunctionAwaitResolveClosure [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
90: 0x105261cbc Builtins_PromiseFulfillReactionJob [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
91: 0x1051d9e38 Builtins_RunMicrotasks [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
92: 0x1051b73c8 Builtins_JSRunMicrotasksEntry [/Users/richard/.nvm/versions/node/v14.16.1/bin/node]
93: 0x150008000 
✨  Done in 2.69s.
```

### System

- Node.js version: 14.16.1
- NPM version: 6
- Strapi version: 4
- Database: Sqlite3
- Operating system: MacOS BigSur 11.6

**Comment by Webbist-dev** [comment]
Use `yarn add @strapi/plugin-graphql` to install it manually

**Comment by kasonde** [comment]
Thanks @Webbist-dev also, fixed in v4.0.5
